### ISSUE TITLE
Shiftit not resizing OSX Finder windows, applies commands to background Chrome windows instead

### ISSUE THREAD

**Issue by stomatocode** [fikovnik/ShiftIt#251]
Have Chrome in background, bring Finder to focus with cmd+tab, use hotkeys to resize Finder windows as I always have; instead the commands are applying to the Chrome windows in the background and ignore Finder windows.

**Comment by stomatocode** [comment]
Update: this actually applies to any app that is technically in the foreground/in focus; Chrome is always the target of the keyboard shortcuts now, no matter what.

**Comment by fikovnik** [comment]
Did you tried the latest beta from releases?

**Comment by isolovey-robarts** [comment]
Happens intermittently to me as well, whether I compile the latest release or the master branch.

**Comment by isolovey-robarts** [comment]
This issue occurs on `version-1.6.4` as well.

**Comment by afinne** [comment]
Noticed this the first time a couple of weeks ago. I haven't updated my OS or Shiftit in a long time, so I assume that a Chrome update made this happen.

**Comment by afinne** [comment]
A "workaround" that works for me at least is to kill Chrome, and when starting it up again, Shiftit works as it should for a while at least.

**Comment by fikovnik** [comment]
Interesting issue. I lost my apple laptop and now waiting for a new one. I will look into this as soon as it arrives.

**Comment by eMiami** [comment]
ANy update on this?  This app has become unusable, all it does is control Chrome.

**Comment by isolovey-robarts** [comment]
Clone and build in Xcode. Problem disappears.

**Comment by isolovey-robarts** [comment]
```
git clone https://github.com/fikovnik/ShiftIt
git checkout version-1.6.4
open ShiftIt/ShiftIt.xcodeproj
```

- `Product>Build For>Running`
- in the file browser pane (left), right-click on `Products/ShifIt.app`, `Reveal in Finder`, then copy to `/Applications`.

**Comment by eMiami** [comment]
thx.  i'm sorry but i don't follow.  should i paste those 3 lines starting with git into terminal?

**Comment by isolovey-robarts** [comment]
Yes. It'll clone this repo at the right commit and open Xcode. The rest is done in the Xcode IDE (GUI).

**Comment by thekashifmalik** [comment]
Just ran into this issue myself. Very anooying. Don't think I'm gonna compile it right now, maybe later. 

Would be nice to have a fix in the distribution.
